tests/functional/examples/market_maker/test_on_chain_market_maker.py

```diff
@@ -31,25 +31,21 @@ def test_initial_state(market_maker):
     assert market_maker.owner() is None
 
 
-def test_initiate(w3, market_maker, erc20, assert_tx_failed):
+def test_initiate(w3, market_maker, erc20, tx_failed):
     a0 = w3.eth.accounts[0]
-    erc20.approve(market_maker.address, w3.to_wei(2, "ether"), transact={})
-    market_maker.initiate(
-        erc20.address, w3.to_wei(1, "ether"), transact={"value": w3.to_wei(2, "ether")}
-    )
-    assert market_maker.totalEthQty() == w3.to_wei(2, "ether")
-    assert market_maker.totalTokenQty() == w3.to_wei(1, "ether")
+    ether, ethers = w3.to_wei(1, "ether"), w3.to_wei(2, "ether")
```

Comment:
maybe just have a constant `ether = w3.to_wei(1, "ether")` and compute `1 * ether` and `2 * ether` in the below lines. that or rename to: `one_ether, two_ether`